test/System.Web.Razor.Test/Text/LookaheadTextReaderTestBase.cs

```diff
@@ -159,7 +159,9 @@ protected void RunReadUntilTest(Func<LookaheadTextReader, string> readMethod, in
             Assert.Equal(expectedRaw, actualLocation.AbsoluteIndex);
             Assert.Equal(expectedChar, actualLocation.CharacterIndex);
             Assert.Equal(expectedLine, actualLocation.LineIndex);
-            Assert.Equal('b', reader.Peek());
+
+            // Workaround xunit/xunit#1502
+            Assert.Equal('b' - '\0', reader.Peek());
```

Comment:
This is really gross. Should we just disable the affected analyzer in the meantime? We should not change our code to work around bugs in unnecessary components...

Reply:
Tried suppressing the analyzer that's blowing up due to xunit/xunit#1502 but unfortunately it still runs far enough to hit the NRE before noticing the rule is disabled. Could either leave this alone or disable the xUnit analyzers entirely in this project. This would be temporary in any case, lasting only until we address #96.

Which option do you prefer?

Reply:
I'd rather not have the analyzer because we don't super need that, and we really don't want this ugly code.